aptos-move/framework/aptos-framework/sources/atomic_bridge.move

```diff
@@ -822,6 +828,34 @@ module aptos_framework::atomic_bridge_store {
         keccak256(combined_bytes)
     }
 
+    #[view]
+    public fun get_bridge_transfer_details_initiator(
+        bridge_transfer_id: vector<u8>
+    ): BridgeTransferDetails<address, EthereumAddress> acquires SmartTableWrapper {
+        let table = borrow_global<SmartTableWrapper<vector<u8>, BridgeTransferDetails<address, EthereumAddress>>>(@aptos_framework);
+
+        let details_ref = smart_table::borrow(
+            &table.inner,
+            bridge_transfer_id
+        );
+
+        *details_ref
+    }
+
+    #[view]
+    public fun get_bridge_transfer_details_counterparty(
+        bridge_transfer_id: vector<u8>
+    ): BridgeTransferDetails<EthereumAddress, address> acquires SmartTableWrapper {
+        let table = borrow_global<SmartTableWrapper<vector<u8>, BridgeTransferDetails<EthereumAddress, address>>>(@aptos_framework);
+
+        let details_ref = smart_table::borrow(
+            &table.inner,
+            bridge_transfer_id
+        );
+
+        *details_ref
+    }
+
```

Comment:
Would this be clearer?  Would need documentation around this I think.

```rust
    #[view]
    public fun get_bridge_transfer_details_initiator(
        bridge_transfer_id: vector<u8>
    ): BridgeTransferDetails<address, EthereumAddress> acquires SmartTableWrapper {
        get_bridge_transfer_details(bridge_transfer_id)
    }

    #[view]
    public fun get_bridge_transfer_details_counterparty(
        bridge_transfer_id: vector<u8>
    ): BridgeTransferDetails<EthereumAddress, address> acquires SmartTableWrapper {
        get_bridge_transfer_details(bridge_transfer_id)
    }

    fun get_bridge_transfer_details<Initiator, Recipient>(bridge_transfer_id: vector<u8>
    ): BridgeTransferDetails<Initiator, Recipient> acquires SmartTableWrapper {
        let table = borrow_global<SmartTableWrapper<vector<u8>, BridgeTransferDetails<Initiator, Recipient>>>(@aptos_framework);

        let details_ref = smart_table::borrow(
            &table.inner,
            bridge_transfer_id
        );

        *details_ref
    }
```

Reply:
Updated, yep definitely clearer. 

You mean documentation about how to call the view functions? Or on using the bridge modules in general?